packages/forms/resources/views/components/textarea.blade.php

```diff
@@ -34,21 +34,21 @@
         "
     >
         <textarea
+            @if (FilamentView::hasSpaMode())
+                ax-load="visible"
+            @else
+                ax-load
+            @endif
+            ax-load-src="{{ \Filament\Support\Facades\FilamentAsset::getAlpineComponentSrc('textarea', 'filament/forms') }}"
+            x-data="textareaFormComponent({ initialHeight: @js($initialHeight) })"
             @if ($shouldAutosize)
-                @if (FilamentView::hasSpaMode())
-                    ax-load="visible"
-                @else
-                    ax-load
-                @endif
-                ax-load-src="{{ \Filament\Support\Facades\FilamentAsset::getAlpineComponentSrc('textarea', 'filament/forms') }}"
-                x-data="textareaFormComponent({ initialHeight: @js($initialHeight) })"
-                x-intersect.once="render()"
-                x-on:input="render()"
-                x-on:resize.window="render()"
-                {{ $getExtraAlpineAttributeBag() }}
+                x-intersect.once="onResize()"
+                x-on:input="onResize()"
+                x-on:resize.window="onResize()"
             @endif
-            x-ignore
-            wire:ignore.style.height
+            x-on:mouseup="onResize()"
```

Comment:
Does this need to be part of the `@if` block above?

Reply:
Oh my bad, i think its actually redundant, can be removed

Reply:
Removed in new commit